Entities/MOSRotating.cpp

```diff
@@ -1591,18 +1594,23 @@ void MOSRotating::Update() {
 
 bool MOSRotating::DrawMOIDIfOverlapping(MovableObject *pOverlapMO)
 {
-    if (pOverlapMO != this && m_GetsHitByMOs)
-    {
-        float combinedRadii = GetRadius() + pOverlapMO->GetRadius();
-        Vector otherPos = pOverlapMO->GetPos();
+    if (pOverlapMO == this || !m_GetsHitByMOs || !pOverlapMO->GetsHitByMOs()) {
```

Comment:
Pretty sure this additional check for pOVerlapMO->GetsHitByMOs() is actually a bug. But it doesn't matter, because this entire function is now unused and will be deleted later. These changes here were optimizations to the redraw that are now wholly unnecessary as MOID redrawing is no longer a thing.

Reply:
I’m not certain it’s a bug, it’s just that the intricacies of GetsHitByMOs and HitsMOs means that things aren’t always as clear as they ought to be. In this case, `this` might not be hit by MOs, but still HitsMOs, in which case the redrawing should occur.

But yeah. It’s completely irrelevant and this redrawing code is generally a travesty. It’s deleted in the multithreaded-sim-and-render branch.